.github/workflows/release-bump-version.yml

```diff
@@ -0,0 +1,87 @@
+name: Release - Bump Version
+
+on:
+  workflow_dispatch:
+    inputs:
+      version_type:
+        description: "Version Type?"
+        required: true
+        type: choice
+        options:
+          - major
+          - minor
+          - patch
+        default: "minor"
+
+jobs:
+  Create-PR-To-Bump-Fetch-Metadata-Version:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          # Ensure we start from main in case the workflow is run from a branch
+          ref: "main"
+          token: ${{ secrets.DEPENDABOT_AUTOMATION_PAT }}
+
+      - name: Read .nvmrc
+        id: nvm
+        run: echo ::set-output name=NVMRC::$(cat .nvmrc)
```

Comment:
This throws a deprecation warning about `set-output`, but that's currently true of three of the other workflows in this repo... for example: https://github.com/dependabot/fetch-metadata/blob/b3bc7993c0c25ee946279581daedf3fbefc2909a/.github/workflows/dependabot-build.yml#L45

So probably simplest to address in a separate PR.